airflow/api_fastapi/execution_api/routes/task_instances.py

```diff
@@ -237,7 +237,7 @@ def ti_heartbeat(
 )
 def ti_put_rtif(
     task_instance_id: UUID,
-    put_rtif_payload: RTIFPayload,
```

Comment:
I just removed `RTIFPayload` and `RootModel` -- we can just have it directly over here

Reply:
ACK. Sounds good